spec/features/support/errors_in_form.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples 'errors in form' do
+  it 'shows there is a problem' do
+    expect(page).to have_content 'There is a problem'
+  end
+
+  it 'adds the error css class' do
+    expect(page).to have_css 'div.govuk-form-group--error'
```

Comment:
@adrianclay Because the style is part of the Gov.uk requirements, I wanted to have it tested. Its part of the acceptance criteria. But generally I agree with you. (This is the red left margin on forms FYI)